src/services_models/reserves.jl

```diff
@@ -44,13 +54,12 @@ function service_requirement_constraint!(
         container = get_parameter(
             container,
             RequirementTimeSeriesParameter(PSY.Deterministic, "requirement"),
-            SR,
+            SR;
+            meta = name,
         )
         param = get_parameter_array(container)
         multiplier = get_multiplier_array(container)
         for t in time_steps
-            param[name, t] = add_parameter(optimization_container.JuMPmodel, ts_vector[t])
-            multiplier[name, t] = requirement
             if use_slacks
```

Comment:
The slacks also need to be refactored properly like the network slacks.

Reply:
that would require to create expression for the reserve balance constraint, and then implement a `add_to_expression` right ?